runtime/service_integ_test.go

```diff
@@ -1546,6 +1547,35 @@ func TestStopVM_Isolated(t *testing.T) {
 				}
 			},
 		},
+
+		{
+			name:       "Unsuccessful",
+			withStopVM: true,
+
+			createVMRequest: proto.CreateVMRequest{},
+			stopFunc: func(ctx context.Context, fcClient fccontrol.FirecrackerService, req proto.CreateVMRequest) {
```

Comment:
more of an open-ended q but curious if we wanted to start assigning the funcs a var name.

imo, it would be useful for readability but unless we can eke out some reuse i am not too partial to any option.